package.json

```diff
@@ -29,7 +29,10 @@
     "url": "https://github.com/cubing/icons.git/issues"
   },
   "description": "Cubing Icons and Fonts",
-  "files": ["dist", "ts"],
+  "files": [
```

Comment:
@jfly: This is the fight between Biome and `npm version` I mentioned. I think Biome's formatting is better, but `npm` is the canonical tool for package publication. Not sure what the best option is here.

Reply:
Gotcha. I'm happy with:

1. Ignoring `package.json`
2. OR: adding a postversion that formats package.json

Reply:
Alright, I'll stick with ignoring.

`postversion` might be a good start, but there are various reasons tools might rewrite `package.json` and I don't have a full list anywhere.